### PR TITLE
Improved: "Mark an article as read…" text area. Added a link to the documentation

### PR DESCRIPTION
**Pull request by math-GH**
Inspired by #5346

Added a link to the documentation in the help text.

After:
grafik

Changes proposed in this pull request:

- i18n

How to test the feature manually:

1. see feed configuration

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/i18n/fr/sub.php [hunk 1]
```diff
@@ -68,7 +68,7 @@
 		'error' => 'Ce flux a rencontré un problème. Veuillez vérifier qu’il est toujours accessible puis actualisez-le.',
 		'filteractions' => array(
 			'_' => 'Filtres d’action',
-			'help' => 'Écrivez une recherche par ligne.',
+			'help' => 'Écrivez une recherche par ligne. Operators <a href="https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field" target="blank">see documentation</a>.',	// DIRTY
```
Comment (Alkarex) [inline app/i18n/fr/sub.php]:
```suggestion
			'help' => 'Écrivez une recherche par ligne. Voir la <a href="https://freshrss.github.io/FreshRSS/fr/users/03_Main_view.html#gr%C3%A2ce-au-champ-de-recherche" target="_blank">documentation des opérateurs</a>.',
```

File: app/i18n/en/sub.php [hunk 2]
```diff
@@ -68,7 +68,7 @@
 		'error' => 'This feed has encountered a problem. Please verify that it is always reachable then update it.',
 		'filteractions' => array(
 			'_' => 'Filter actions',
-			'help' => 'Write one search filter per line.',
+			'help' => 'Write one search filter per line. Operators <a href="https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field" target="blank">see documentation</a>.',
```
Comment (Alkarex) [inline app/i18n/en/sub.php]:
```suggestion
			'help' => 'Write one search filter per line. Operators <a href="https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field" target="_blank">see documentation</a>.',
```

And the same for all other translations

**Comment by Alkarex** [comment]
Sorry for the slow review; this PR went under my radar...

File: app/i18n/en/sub.php (hunk 2 above)
Comment (math-GH) [reply app/i18n/en/sub.php]: oh no :/

Comment (math-GH) [reply app/i18n/en/sub.php]: fixed

**Pull request merged**